postal/service.go

```diff
@@ -237,15 +254,28 @@ func (s Service) Deliver(dependency Dependency, cnbPath, layerPath, platformPath
 		dependencyChecksum = fmt.Sprintf("sha256:%s", dependency.SHA256)
 	}
 
+	dependencyMirrorURI, err := s.mirrorResolver.FindDependencyMirror(dependency.URI, platformPath)
+	if err != nil {
+		return fmt.Errorf("failure checking for dependency mirror: %s", err)
+	}
+
 	dependencyMappingURI, err := s.mappingResolver.FindDependencyMapping(dependencyChecksum, platformPath)
 	if err != nil {
 		return fmt.Errorf("failure checking for dependency mappings: %s", err)
 	}
 
+	if dependencyMirrorURI != "" && dependencyMappingURI != "" {
+		return fmt.Errorf("dependency mappings should not exist when using a dependency mirror")
```

Comment:
what's your thinking in having this error out, over doing something like choosing a mirror to have precedence over a mapping?

Reply:
Regardless we should do whatever they're doing in libpak so the experience is consistent

Reply:
Hmmm...I think I might have made this up 🙃 It's not in the RFC and I dont see it in libpak so I'll remove it

Reply:
This check also exists in [libpack](https://github.com/bitgully/libpak/blob/main/dependency_cache.go#L257).

Reply:
Bump on this one - I see the lines commented out around this. Can you get rid of that since we've decided against the logger?